webpack.config.js

```diff
@@ -3,6 +3,8 @@ const path = require('path');
 const SRC_DIR = path.join(__dirname, '/client/src');
 const DIST_DIR = path.join(__dirname, '/client/dist');
 
+console.log('DIST_DIR IS ', DIST_DIR);
```

Comment:
remove console log unless you want to keep for further development

Reply:
done. actually, not sure that line works.